add-github-organization.sh

```diff
@@ -4,6 +4,8 @@ set -ex
 gh repo list $1 --language java --no-archived --source --visibility public --limit 1000 --json nameWithOwner,defaultBranchRef --template "{{range .}},{{.nameWithOwner}},{{.defaultBranchRef.name}},,,,,,,{{\"\n\"}}{{end}}" > new.csv
 gh repo list $1 --language python --no-archived --source --visibility public --limit 1000 --json nameWithOwner,defaultBranchRef --template '{{range .}},{{.nameWithOwner}},{{.defaultBranchRef.name}},,,,,,,{{"\n"}}{{end}}' >> new.csv
 
+gh search repos --language javascript --sort stars --visibility public --limit 1000 --json fullName,defaultBranch --template "{{range .}},{{.fullName}},{{.defaultBranch}},,,,,,,{{\"\n\"}}{{end}}"  > new.csv
```

Comment:
Did you mean to commit this. The change will override the csv

Reply:
revised